### ISSUE TITLE
Google Colab File Upload Does Not Work in Firefox

### ISSUE THREAD

**Issue by AlexandraGoffova** [googlecolab/colabtools#3479]
**Describe the current behavior**
When trying to upload files in Google Colab while using Firefox the loading circle keeps spinning and turns red after some time. Then the file fails to upload in the folder.

**Describe the expected behavior**
The loading circle should display the status of the file being uploaded and when finished this file should appear in the desired folder. This works correctly in Google Chrome, however when using Firefox this actions fails to work as expected.

**What web browser you are using?**
(Chrome, Firefox, Safari, etc.)
Firefox

**Additional context**
The file being uploaded is a '.pb' file.

**Comment by cperry-goog** [comment]
How big is your file?

**Comment by AlexandraGoffova** [comment]
> How big is your file?

The file is 48.8 MB

**Comment by AlexandraGoffova** [comment]
> Does this work better for you as well?

Yes it works when doing it via code, I opened this issue just to highlight that it doesn't work manually.

Thanks for the advice !

**Comment by youdontneedtoknow22** [comment]
lots of comments for reproducing are being mentioned here
https://github.com/googlecolab/colabtools/issues/423#issuecomment-1687769284

**Comment by nikitakor7** [comment]
Having the same issue on google chrome now!
